src/view/components/Drawer/DrawerContents/index.tsx

```diff
@@ -1,22 +1,31 @@
 import * as React from 'react';
 import { Content, List, Button, Text } from 'native-base';
-import { routes, RouteInfo, Route } from '../../../navigation/navigator';
+
+import { routesInfo, RouteInfo, Route, DrawerRouteInfo } from '../../../navigation/navigator';
 import style from './style';
 import DrawerItem from '../DrawerItem';
 import DrawerHeader from '../DrawerHeader';
 
 /** DrawerContents props */
-export interface OwnProps {
-  onPress(route: Route): void;
+export interface OwnPropsType {
+  name: string;
+  username: string;
+  onPress: OnPress;
   onLogout(): void;
```

Comment:
not related but do you think we need to make log out work for the demo/prototype?

Reply:
I think the biggest risk of it not working is us forgetting that we didn't actually log them out